packages/material/scss/treelist/_layout.scss

```diff
@@ -1,6 +1,8 @@
 @import "@progress/kendo-theme-default/scss/treelist/_layout.scss";
 
-@include exports("treelist/layout/material") {
+
+@mixin kendo-treelist--layout() {
+
```

Comment:
missing @include kendo-treelist--layout-base(); here. I guess this will also fix the differences in the visual tests

Reply:
thank you!